src/common/plugin.c

```diff
@@ -1521,7 +1549,11 @@ hexchat_list_int (hexchat_plugin *ph, hexchat_list *xlist, const char *name)
 {
 	guint32 hash = str_hash (name);
 	gpointer data = ph->context;
-	int tmp = 0;
+
+    int channel_flag;
+	int channel_flags[CHANNEL_FLAG_COUNT];
+	int channel_flags_flag = 0;
```

Comment:
These names are a bit easy to confuse, no better suggestion off the top of my head.

Reply:
I change to used